psi4/src/export_cubeprop.cc

```diff
@@ -39,6 +40,8 @@ namespace py = pybind11;
 void export_cubeprop(py::module& m) {
     py::class_<CubeProperties, std::shared_ptr<CubeProperties>>(m, "CubeProperties", "docstring")
         .def(py::init<std::shared_ptr<Wavefunction>>())
+        .def("compute_density", &CubeProperties::compute_density)
+        .def("compute_orbitals", &CubeProperties::compute_orbitals)
```

Comment:
go ahead and add `"D"_a, "key"_a` and `"C"_a, "indices"_a, "labels"_a, "key"_a`, respectively

Reply:
Done.